cypress/component/LibraryCard/LibraryCardTable.spec.tsx

```diff
@@ -96,7 +96,7 @@ describe('Library Card Table Tests', () => {
 
     mount(<LibraryCardTable {...props} />);
 
-    cy.get('[data-cy=search-bar]').type(libraryCardList[0].userName.toLowerCase());
+    cy.get('[data-cy=search-bar]').type(libraryCardList[0].userName);
```

Comment:
not entirely sure why these dependency updates caused this test to start failing, but the underlying search functionality leans into lodash/fp a lot, so i'm assuming it's something related to that particular update